src/public/modules/forms/admin/directives/validate-email-domain-from-text.directive.js

```diff
@@ -1,5 +1,7 @@
 'use strict'
 
+const { validateEmailDomains } = require('shared/util/email-domain-validation')
```

Comment:
does this import not fail? :O it's not relative

Reply:
Thanks for pointing this out. I saw the same in `src/public/modules/forms/services/file-handler.client.service.js` and assumed this will work too. But there are also many other require calls to `shared/util` which are relative paths containing `../`. Will fix this.

Reply:
Oh, you are right! Maybe there's some black magic behind the scenes for shared...... Nonetheless, thanks!